providers/implementations/serializers/serializer_common.c

```diff
@@ -243,6 +243,36 @@ int ossl_prov_print_labeled_bignum(BIO *out, const char *label,
     return 1;
 }
 
+/* Number of octets per line */
+#define LABELED_BUF_PRINT_WIDTH    15
```

Comment:
Why do we use such a weird number, why not 16??

Reply:
To be consistent with the way `ASN1_buf_print` currently does this.